pkg/cmd/roachtest/tests/versionupgrade.go

```diff
@@ -403,6 +413,18 @@ func enableTracingGloballyStep(ctx context.Context, t test.Test, u *versionUpgra
 	}
 }
 
+// lowerLeaseDuration sets the `sql.catalog.descriptor_lease_duration`
+// setting to 1 minute.
+func lowerLeaseDuration(node int) versionStep {
+	return func(ctx context.Context, t test.Test, u *versionUpgradeTest) {
+		db := u.conn(ctx, t, node)
+		_, err := db.ExecContext(ctx, `SET CLUSTER SETTING sql.catalog.descriptor_lease_duration = '00:01:00'`)
```

Comment:
Nit, duration format [1] might be easier on the eye, e.g.,

```
SET CLUSTER SETTING kv.closed_timestamp.target_duration = '20s'
```

[1] https://www.cockroachlabs.com/docs/stable/interval.html#duration-fields

Reply:
Makes sense, updated 👍